lab_1/main.py

```diff
@@ -36,7 +53,14 @@ def calculate_frequencies(tokens: list) -> dict:
     e.g. tokens = ['weather', 'sunny', 'man', 'happy']
     --> {'weather': 1, 'sunny': 1, 'man': 1, 'happy': 1}
     """
-    pass
+    if isinstance(tokens, list):
+        dictionary = {}
+        frequency = 0
```

Comment:
There is no need to define this variable, you can just do it the the `for`-loop.